plugins/hydra_ray_launcher/examples/docker/config.yaml

```diff
@@ -0,0 +1,48 @@
+task: 1
+
+hydra:
+  launcher:
+    _target_: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
+    env_setup:
+      pip_packages:
+        omegaconf: null
+        hydra_core: null
+        ray: null
+        cloudpickle: null
+        pickle5: null
+        hydra_ray_launcher: null
+      commands: []
+    ray:
+      cluster:
+        cluster_name: example-cluster
+        min_workers: 0
+        max_workers: 0
+        initial_workers: 0
+        autoscaling_mode: default
+        target_utilization_fraction: 0.8
+        idle_timeout_minutes: 5
+        docker:
+          image: 'samuelstanton/ray-plugin-example'
```

Comment:
could we use a public docker image, maybe one from the ray team's examples?

Reply:
this image is public :). I don't think you can use any old image because you need hydra and its dependencies installed. 

https://hub.docker.com/repository/docker/samuelstanton/ray-plugin-example